manifests/client/params.pp

```diff
@@ -3,16 +3,10 @@
 # @api private
 #
 class mongodb::client::params inherits mongodb::globals {
-  $package_ensure = pick($mongodb::globals::version, 'present')
+  $package_ensure = pick($mongodb::globals::client_version, 'present')
   $manage_package = pick($mongodb::globals::manage_package, $mongodb::globals::manage_package_repo, false)
 
-  if $manage_package {
-    $package_name = "mongodb-${mongodb::globals::edition}-shell"
-  } else {
-    $package_name = $facts['os']['family'] ? {
-      'Debian' => 'mongodb-clients',
-      'Redhat' => "mongodb-${mongodb::globals::edition}-shell",
-      default  => 'mongodb',
-    }
-  }
+  # the new mongosh package is the same for all distros.
+  # and it follows its own versioning
+  $package_name = 'mongodb-mongosh'
```

Comment:
If users have local scripts or puppet code using shell this change will break that...
I think we should make the client class install both both in the beginning.

Reply:
I tend to agree for versions < 6.0

Reply:
I discussed this with @witjoh (he's currently sitting right across me) and he has a point. Let's not bother with this, we already indicate we have breaking changes.

Reply:
I honestly don't understand why you want to push such a break.
The mongo command was deprecated in 5.0 and is a valid command for another 7 months.
The only thing we have to do is add 10 lines of code to install the package.
We don't have to support the command in providers as that is internal module code.

Reply:
Users can still add a package resource themselves. If we wouldn't have any other breaking changes, I'd agree, but since we already do, what's the point of adding deprecated commands to new installations?

Reply:
See your point. I'm not going to hold back on approval because of this even if I don't agree.

Reply:
FYI for anyone who _does_ need to support both mongo < 4.4 and > 5.0 ( sadly that's me ) then you can always add a simple mongo version check to the respective mongodb.rb files and have an IF ELSE statement with both the old code and the new :-) ( yes....it's not ideal....but hey it works )